src/lib/viewers/archive/Archive.scss

```diff
@@ -0,0 +1,6 @@
+.bp-archive {
```

Comment:
It seems like overkill to make a separate HTTP call for this small amount of CSS. I wonder if we can just inline it in the React component?

Reply:
> It seems like overkill to make a separate HTTP call for this small amount of CSS. I wonder if we can just inline it in the React component?

The final css file also contains all the css code from the renderers.